docs/src/prototypes/chatPane/services/messageFactoryMock.tsx

```diff
@@ -44,6 +46,53 @@ function generateChatMsgProps(msg: MessageData, fromUser: UserData): ChatMessage
   return msgProps
 }
 
+function getMessageContent(content: string) {
```

Comment:
In fact, this is a content factory, may be other name?

```suggestion
function createMessageContent(content: string) {
```